### PR TITLE
docs(README): update required parameters

### LINKED ISSUE #115: Change current module parameter to required
For consistency in the solution center navigation UI, the `current-module` parameter should be required for all products within the solution center (i.e. not the solution center itself). We should update the README to reflect this.

### PR DESCRIPTION
**Pull request by cjhowald**
Fixes #115 

Note: also promoted user & modules to required since current-module relies on modules and the header won't work without a user.

### CODE REVIEW

**Comment by gregdeane** [comment]
Everything looks good. Please address the one comment.

File: README.md [hunk 1]
```diff
@@ -31,49 +31,64 @@ npm install solution-center-header
...
 
- - **current-module**: A number representing the id of the current module displayed, in order to highlight it in the header
+ - **solution-center-url**:  A URL override for the solution center. This URL is used to construct the links for the user account, brand account, and brand switcher pages. The URL should have a trailing `/#/`. If no URL is provided, the solution center production URL is used by default.
 
-    ```current-module = 0```
+    ```javascript
+    solution-center-url = "https://www.example.com/#/"
+    ```
     
  - **debug-config: Configuration object for debugging purposes
```
Comment (cjhowald) [inline README.md]: Fixed

**Comment by gregdeane** [comment]
👍

**Comment by cjhowald** [comment]
👍

**Pull request merged**